### PR TITLE
Whistleblower

### PR DESCRIPTION
**Pull request by tekram**

### CODE REVIEW

File: source/sections/19-employees_policy.md [hunk 1]
```diff
@@ -48,14 +48,14 @@ Luma is committed to ensuring all workforce members actively address security an
 
 ##  19.3 Issue Escalation
 
-Luma workforce members are to escalate issues using the procedures outlined in the Employee Handbook. Issues that are brought to the Escalation Team are assigned an owner. The membership of the Escalation Team is maintained by the Chief Executive Officer.
+Luma workforce members are to escalate issues using the procedures outlined in the Employee Handbook. Issues that are brought to the Escalation Team are assigned an owner. The membership of the Escalation Team is maintained by the Chief Executive Officer. The eomployee can do so via an anonymous form: https://forms.gle/8ZTWwTSdYwyq5wws5
```
Comment (demon) [inline source/sections/19-employees_policy.md]: `s/eomployee/employee/`

This whole paragraph is hard to read. Might I suggest:

> Luma workforce members are to escalate issues using the procedures outlined in the Employee Handbook, and can do so via [an anonymous form](https://forms.gle/8ZTWwTSdYwyq5wws5). Issues that are raised are brought to the Escalation Team and assigned an owner. The membership of the Escalation Team is maintained by the Chief Executive Officer.

File: source/sections/19-employees_policy.md [hunk 2]
```diff
@@ -48,14 +48,14 @@ Luma is committed to ensuring all workforce members actively address security an
...
 
 It is the duty of that owner to follow the process outlined below:
 
-1. Create an Issue in the Asana Compliance Review Activity (CRA) Project.
+1. Create an Issue in the Click Up Compliance Review Activity (CRA) Project.
 2. The Issue is investigated, documented, and, when a conclusion or remediation is reached, it is moved to Review.
 3. The Issue is reviewed by another member of the Escalation Team. If the Issue is rejected, it goes back for further evaluation and review.
 4. If the Issue is approved, it is marked as Done, adding any pertinent notes required.
-5. The workforce member that initiated the process is notified of the outcome via email.
+5. The workforce member that initiated the process is notified of the outcome via email or Slack. If it anonymous, this will be comminuacted more boradly if deemed by the Escalation Team to be needed.
```
Comment (demon) [inline source/sections/19-employees_policy.md]: `s/boardly/broadly/` and `s/comminuacted/communicated/`

Generally I'd just stick to email so there's a clear paper trail.

File: source/sections/19-employees_policy.md (hunk 1 above)
Comment (tekram) [reply source/sections/19-employees_policy.md]: It does read better lets do that.

File: source/sections/19-employees_policy.md (hunk 2 above)
Comment (tekram) [reply source/sections/19-employees_policy.md]: Sure. That works for me.

Comment (tekram) [reply source/sections/19-employees_policy.md]: updated

File: source/sections/19-employees_policy.md (hunk 1 above)
Comment (tekram) [reply source/sections/19-employees_policy.md]: updated

**Pull request merged**